test/test_nn.py

```diff
@@ -9810,26 +9810,28 @@ def helper(n, c, h, w, kernel_size, stride=None,
         helper(10, 512, 31, 31, 3, stride=2)
         helper(1, 129, 8, 8, 3, stride=2)
 
-    @onlyCUDA
```

Comment:
why do you think this was onlyCUDA before?  Isn't your test (on CPU) going to run the same thing twice and check it's the same?  That's fine I guess?

Reply:
The original purpose of this onlyCUDA was to test CUDA implementation with CPU impl as a reference. I would also suggest that we keep onlyCUDA there, otherwise there would be a duplicate CPU-CPU comparison. 

In order to have a "purely CPU" test, we can have a few hard-coded pooling input and results.